### DISCUSSION TITLE
change the size of the `/` partition in the default layout?

### DISCUSSION THREAD

**Discussion by amtoine** [archlinux/archinstall#1594] [Q&A]
i was looking at [this video](https://youtu.be/vUKRFyI3uPs) and it reminded me of a problem i had with my system not so long ago :yum: 

i once installed Arch with `archinstall` on my 256Gib hard drive, with the default layout and a separate `/home` partition :+1: 
however, the default layout gave me only 20Gib for the `/` partition and it got full very quickly, and that was quite painfull to resize (the position of the partition, the order of the partitions and the fact that it was encrytped), so i fresh installed Arch again :confused: 

## my question
is there a way, from within `archinstall`, to change the size of the `/` partition in the default layout with separate `/home` without going full manual partitioning? :open_mouth: 
i  would love to know :yum:

**Comment by svartkanin** [comment]
At this point no there isn't.
However, in the manual partitioning there's the option to suggest a layout as well which will create the same partitions as you experienced.
After that you can simply adjust the sizes of each and leave everything else as is

**Reply by amtoine** [reply]
mm ok i see :+1: 

i did not have the time to test that, i'll probably mark that the answer as soon as i have the opportunity to try it out :relieved: :yum:
